src/ui/views/ViewSubscriptionNeeded.qml

```diff
@@ -170,21 +170,31 @@ VPNFlickable {
                 }
             }
 
+            VPNFooterLink {
```

Comment:
Let's use `VPNLinkButton` here or we'll have to set all of the anchor properties in `VPNFooterLink` to 'undefined'.